src/shared/components/query/QueryContainer.tsx

```diff
@@ -202,11 +203,13 @@ export default class QueryContainer extends React.Component<
                         </div>
                     )}
 
-                {this.store.unknownStudyIds.isComplete && (
-                    <UnknownStudiesWarning
-                        ids={this.store.unknownStudyIds.result}
-                    />
-                )}
+                {this.store.unknownStudyIds.isComplete &&
+                    getServerConfig()
+                        .skin_home_page_show_unauthorized_studies === false && (
```

Comment:
```suggestion
                    !!getServerConfig()
                        .skin_home_page_show_unauthorized_studies && (
```

Reply:
This also takes into account that the value is not defined.

Reply:
Done